index.js

```diff
@@ -229,14 +229,21 @@ RadialProgressChart.prototype.update = function (data) {
 
   var center = self.svg.select("text.rbc-center-text");
 
+  var delay = self.options.animation.delay;
+  var duration = self.options.animation.duration;
+
+  if(self.options.animation.disable) {
```

Comment:
If the disable option sets the delay and duration both to 0, can you achieve the same (unanimated) results with using the current library and passing in 0s to both options.duration and options.delay?

Reply:
In the `normalizeOptions` function, it checks if the values exist before defaulting to some values. I didn't want to remove the check because I don't think all calls of the graphs that are used throughout the site supply delay/duration values so it would cause all of those to have no animations. I'll change it to explicity look for undefined instead though.